Dockerfile

```diff
@@ -79,22 +125,36 @@ RUN pip3 install snakemake==5.10.0
 # from Google Storage URIs.
 RUN pip3 install google-cloud-storage==2.1.0
 
-# epiweeks (for ncov)
+# Install epiweeks (for ncov)
 RUN pip3 install epiweeks==2.1.2
 
-# Add Pangolin and PangoLEARN + deps (for ncov)
+# Install Pangolin and PangoLEARN + deps (for ncov)
 RUN pip3 install git+https://github.com/cov-lineages/pangolin.git@v3.1.17
 RUN pip3 install git+https://github.com/cov-lineages/pangoLEARN.git@2021-12-06
 RUN pip3 install git+https://github.com/cov-lineages/scorpio.git@v0.3.16
 RUN pip3 install git+https://github.com/cov-lineages/constellations.git@v0.1.1
 RUN pip3 install git+https://github.com/cov-lineages/pango-designation.git@19d9a537b9
 
-# Add Nextstrain components
+
+# 4. Add Nextstrain components
 
 # Allow caching to be avoided from here on out by calling
 # docker build --build-arg CACHE_DATE="$(date)"
 ARG CACHE_DATE
 
+# Nextclade/Nextalign are downloaded directly but using the latest version, so
+# they belong after CACHE_DATE.
```

Comment:
I was confused at first because I had seen Nextclade/Nextalign downloaded above, but then realized those are v1 and this is v2. This small change would have helped, I think:

```suggestion
# Nextclade/Nextalign v2 are downloaded directly but using the latest version,
# so they belong after CACHE_DATE (unlike Nextclade/Nextalign v1).
```

Reply:
Done: c528ef5cb37a7459ddbfb36efc5cee87eba8757c